tests/test_http.py

```diff
@@ -89,6 +95,7 @@ def test_download_follows_redirect_to_edl_and_adds_auth_headers(
     destination_file = mocker.Mock()
     cfg = config_fixture()
 
+    monkeypatch.setattr(os.path, "getsize", lambda a: 0)
```

Comment:
Since doesn't seem to be not dependent on anything being set up in the test, would it be possible to move this and other copy/paste method calls into a single `setUp` method or similar?  I'm not too familiar with how pytest works in this regard.